src/replica/QservMgtServices.cc

```diff
@@ -346,6 +346,40 @@ GetDbStatusQservMgtRequest::Ptr QservMgtServices::databaseStatus(
     return request;
 }
 
+GetConfigQservMgtRequest::Ptr QservMgtServices::config(std::string const& worker, std::string const& jobId,
+                                                       GetConfigQservMgtRequest::CallbackType const& onFinish,
+                                                       unsigned int requestExpirationIvalSec) {
+    GetConfigQservMgtRequest::Ptr request;
+
+    // Make sure the XROOTD/SSI service is available before attempting
+    // any operations on requests
+
```

Comment:
I don't think this blank line helps anything.

Reply:
Good point!